openedx/adg/lms/webinars/admin.py

```diff
@@ -70,9 +56,24 @@ def webinar_status(self, webinar):
         Returns:
             string: Webinar status either `Upcoming` or `Delivered`
         """
-        if not webinar.start_time:
+        if webinar.is_cancelled:
+            return _('Cancelled')
+        elif not webinar.id or webinar.is_upcoming_webinar:
```

Comment:
Here we are checking for `webinar.id`. Can't we move this check to the `is_upcoming_webinar` property?

Reply:
No, we shouldn't. This check belongs to specific scenario that is to check if we are on create webinar page or not. `is_upcoming_webinar` is a property which must be called every-time with a valid webinar object.